Examples/Resources/Extractors/SAT-features-competition2012_revised_without_SatELite_sparkle/sparkle_extractor_wrapper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""Template for users to create Feature Extractor wrappers."""
+
+import sys
+import os
+import time
+import random
+import argparse
+import subprocess
+from pathlib import Path
+
+global sparkle_special_string
+sparkle_special_string = r'__@@SPARKLE@@__'
```

Comment:
remove

Reply:
Do you know the intention of the string? I also wanted to remove it but it is used in the later part of the program (the part that I'll look into next) so I just left it there

Reply:
Saw it used for creating a selector file once, but honestly not really an idea